src/components/Editor/Board/Canvas/Container.ts

```diff
@@ -71,63 +73,67 @@ export default class Container {
     }
   }
 
-  getMouse(evt: MouseEvent) {
-    this.pointY = evt.pageY - this.offsetY;
-    this.pointX = evt.pageX - this.offsetX;
+  getMouse(evt: MouseEvent): Point {
+    return {
+      y: evt.pageY - this.offsetY,
+      x: evt.pageX - this.offsetX,
+    };
   }
 
   onmousedown(evt: MouseEvent) {
-    this.getMouse(evt);
-    if (this.isOutSide()) {
+    const point = this.getMouse(evt);
+    if (this.isOutSide(point)) {
       return;
     }
 
     dragStatus = DragStatus.Drag;
```

Comment:
`dragStatus` is too open to be in the global scope. How about setting `dragStatus` as a property of `Container`?
```suggestion
    this.dragStatus = DragStatus.Drag;
```